assets/queries/openAPI/2.0/operation_object_parameters_with_body_and_formatdata/query.rego

```diff
@@ -0,0 +1,24 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) == "2.0"
+
+	[path, value] := walk(doc)
+	count(path) != 0
+	param := value.parameters
+	body := [x | p := param[l]; p.in == "body"; x := p.in]
+	formatData := [x | p := param[l]; p.in == "formatData"; x := p.in]
```

Comment:
I think we could use `_` here, since `l` is not used...
```suggestion
	body := [x | p := param[_]; p.in == "body"; x := p.in]
	formatData := [x | p := param[_]; p.in == "formatData"; x := p.in]
```